.github/ISSUE_TEMPLATE/failing_test.yml

```
@@ -0,0 +1,171 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Failing Test
+description: Report a failing test
+title: "[Failing Test]: "
+labels: ["bug", "awaiting triage", "test-failures"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report this test failure! Once you've created an issue, you can self-assign by commenting `.take-issue` and you can close by commenting `.close-issue`.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please name which test is failing.
+      placeholder: |
+          If possible include: when the test started failing, a jenkins link, and any initial debugging you've done.
+
+          If this isn't a bug and you have a question or support request, please email user@apache.beam.com with a description of the problem instead of opening this issue.
+  - type: dropdown
+    id: failure
+    attributes:
+      label: Issue Failure
+      options:
+      - "Test is flaky"
+      - "Test is continually failing"
```

```suggestion
      - "Failure: Test is flaky"
      - "Failure: Test is continually failing"
```

This will allow the bot to pick it up since your config is:

```
- valueFor: 'Failure'
  contains: 'Test is flaky'
  addLabels: ['flaky', 'tests']
- valueFor: 'Failure'
  contains: 'Test is continually failing'
  addLabels: ['permared', 'failing test', 'tests']
```